servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -259,21 +266,25 @@ private String getPrimaryKey(final String provider, final String instanceId, fin
 
         List<X509CertRecord> updatedRecords = new ArrayList<>();
         for (Item item : items) {
-            // For each item, update lastNotifiedTime and lastNotifiedServer (unless they were already updated)
-            UpdateItemSpec updateItemSpec = new UpdateItemSpec().withPrimaryKey(KEY_PRIMARY, item.getString(KEY_PRIMARY))
-                    .withReturnValues(ReturnValue.ALL_NEW)
-                    .withUpdateExpression("set lastNotifiedTime = :lastNotifiedTimeVal, lastNotifiedServer = :lastNotifiedServerVal")
-                    .withConditionExpression("attribute_not_exists(lastNotifiedTime) OR lastNotifiedTime < :v_yesterday")
-                    .withValueMap(new ValueMap()
-                            .with(":lastNotifiedTimeVal", lastNotifiedTime)
-                            .withNumber(":v_yesterday", yesterday)
-                            .withString(":lastNotifiedServerVal", lastNotifiedServer));
-
-            Item updatedItem = table.updateItem(updateItemSpec).getItem();
-
-            if (isRecordUpdatedWithNotificationTimeAndServer(lastNotifiedServer, lastNotifiedTime, updatedItem)) {
-                X509CertRecord x509CertRecord = itemToX509CertRecord(updatedItem);
-                updatedRecords.add(x509CertRecord);
+            try {
```

Comment:
If we'll find a problematic record, we'll print an error and continue so it won't affect other items.